config/index.js

```diff
@@ -70,7 +70,10 @@ module.exports = {
   SERVER_HOST,
   PORT: process.env.PORT || 3000,
   LOG_LEVEL: process.env.LOG_LEVEL || (IS_DEV ? 'debug' : 'error'),
-  NO_CACHE: process.env.CACHE_ASSETS ? false : IS_DEV,
+  STATIC_ASSETS: {
+    CACHE_MAX_AGE:
+      process.env.STATIC_ASSETS_CACHE_MAX_AGE || (IS_PRODUCTION ? '1w' : false),
```

Comment:
@Nimphal not sure what the best production default is but I went with 1 week. It's worth noting we revision any changes so if we changed any of the assets they would be a new URL that the browser wouldn't have previously cached so I felt less concerned about a longer timeframe.